services/app-api/serverless.yml

```diff
@@ -160,6 +160,21 @@ functions:
           method: get
           cors: true
 
+  third_party_api_authorizer:
+    handler: src/handlers/third_party_API_authorizer.main
+
+  jwthealth:
+      handler: src/handlers/health_check.main
+      events:
+        - http:
+            path: health_check
+            method: get
+            cors: true
+            authorizer:
+              name: third_party_api_authorizer
+              identitySource: method.request.header.Authorization 
+              resultTtlInSeconds: 3600 # cache the result for 1 hour
```

Comment:
This is an optional field, not sure if we actually want to cache results now

Reply:
Yeah it seems like it's based on the authorizer token/identity source. I came across [this](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-lambda-authorizer.html#http-api-lambda-authorizer.caching)
https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-lambda-authorizer.html#http-api-lambda-authorizer.caching

I don't fully understand it so I'll remove but if we want to add it back in we can